Neos.ContentRepository.Core/Classes/Projection/Projections.php

```diff
@@ -93,4 +93,8 @@ public function getIterator(): \Traversable
     {
         yield from $this->projections;
     }
+    public function isEmpty(): bool
```

Comment:
missing empty line :)
```suggestion

    public function isEmpty(): bool
```

Reply:
<img width="371" alt="Screenshot 2023-11-29 at 11 43 30" src="https://github.com/neos/neos-development-collection/assets/324408/a7c22ac9-7765-4cd0-bdfe-79c54a202c0d">

git integration let me astray :D